modAionBase/src/org/aion/base/db/IContractDetails.java

```diff
@@ -113,27 +117,19 @@
     byte[] getEncoded();
 
     /**
-     * Returns a mapping of all the key-value pairs who have keys in the collection keys.
+     * Returns a mapping of all the key-value pairs that have keys in the given collection keys.
      *
-     * @param keys The keys to query for.
-     * @return The associated mappings.
+     * @param keys the keys to query for
+     * @return the associated mappings
      */
     Map<ByteArrayWrapper, ByteArrayWrapper> getStorage(Collection<ByteArrayWrapper> keys);
 
-    /**
-     * Sets the storage to contain the specified keys and values. This method creates pairings of
-     * the keys and values by mapping the i'th key in storageKeys to the i'th value in
-     * storageValues.
-     *
-     * @param storageKeys The keys.
-     * @param storageValues The values.
-     */
-    void setStorage(List<ByteArrayWrapper> storageKeys, List<ByteArrayWrapper> storageValues);
-
     /**
      * Sets the storage to contain the specified key-value mappings.
      *
-     * @param storage The specified mappings.
+     * @param storage the specified mappings
+     * @apiNote Used for testing.
```

Comment:
@VisibleForTesting?

Reply:
I marked the fact that it was only used for testing so that maybe it gets removed from the interface during the refactoring. Then the tag can be added to the implementations.